File: docs/breez-sdk/snippets/react-native/fiat_currencies.ts
import { type BreezClient } from '@breeztech/breez-sdk-spark-react-native'

const exampleListCurrencies = async (client: BreezClient) => {
  // ANCHOR: list-fiat-currencies
  const response = await client.listFiatCurrencies()
  // ANCHOR_END: list-fiat-currencies
}

const exampleListRates = async (client: BreezClient) => {
  // ANCHOR: list-fiat-rates
  const response = await client.listFiatRates()
  // ANCHOR_END: list-fiat-rates
}

File: docs/breez-sdk/snippets/kotlin_mpp_lib/shared/src/commonMain/kotlin/com/example/kotlinmpplib/FiatCurrencies.kt
package com.example.kotlinmpplib

import breez_sdk_spark.*

class FiatCurrencies {
    suspend fun listFiatCurrencies(client: BreezClient) {
        // ANCHOR: list-fiat-currencies
        try {
            val response = client.listFiatCurrencies()
        } catch (e: Exception) {
            // handle error
        }
        // ANCHOR_END: list-fiat-currencies
    }

    suspend fun listFiatRates(client: BreezClient) {
        // ANCHOR: list-fiat-rates
        try {
            val response = client.listFiatRates()
        } catch (e: Exception) {
            // handle error
        }
        // ANCHOR_END: list-fiat-rates
    }
}